nuitka/tools/testing/SearchModes.py

```diff
@@ -173,3 +174,15 @@ def abortIfExecuted(self):
         if self.active:
             return True
         return False
+
+
+class SearchModeAll(SearchModeBase):
+    def __init__(self):
+        SearchModeBase.__init__(self)
+        self.total_errors = 0
+
+    def updateTotalErrors(self):
+        self.total_errors += 1
+
+    def finish(self):
```

Comment:
This probably should also `sys.exit()` with error code, so it's noticed by outside world.

Reply:
See however first my comment below.